portfolio/src/main/webapp/mypics/Grunt Webpage.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta charset="UTF-8">
+<title>Adoption In Mass Effect</title>
+<style>

```

Comment:
Can you put these styles in style.css so that this file simply has the html and all the styles are co-located in one file.
Much easier to debug later.